chacha20_poly1305/src/benches.rs

```diff
@@ -0,0 +1,45 @@
+use test::Bencher;
+
+use crate::{ChaCha20, Key, Nonce};
+
+#[bench]
+pub fn chacha20_10(bh: &mut Bencher) {
+  let key =
+      Key::new([0u8; 32]);
+  let nonce = Nonce::new([0u8; 12]);
+  let count = 1;
+  let mut chacha = ChaCha20::new(key, nonce, count);
+  let mut bytes = [0u8; 10];
+  bh.iter(|| {
+    chacha.apply_keystream(&mut bytes[..]);
+  });
+  bh.bytes = bytes.len() as u64;
```

Comment:
Well, you say you're new to Rust but I, after nearly 10 years of Rust experience, had to look up what this line does. :shrug: 

:laughing: